tests/fixtures/phase3_fixtures.py

```
@@ -0,0 +1,562 @@
+        )
+
+    def force_failures(self, enabled: bool = True):
+        """
+        Enable or disable forced failure mode, causing all subsequent calls to this provider to fail if enabled.
+
+        Parameters:
+            enabled (bool): If True, all future calls will fail; if False, normal behavior resumes.
+        """
+        self._forced_failure = enabled
+
+    def _is_error_retryable(self, error):
```

Pylint (3.3.7)

[warning] Unused argument 'error'

(W0613)